### PR TITLE
Documentation on filtering and sharing Kedro-Viz via stateful URLS. 

### PR DESCRIPTION
**Pull request by rashidakanchwala**
## Description

Docs on filtering and sharing via stateful URLS. 

## Checklist

- [ ] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

File: docs/source/share_kedro_viz.md [hunk 1]
```diff
@@ -11,6 +11,16 @@ The publish and share feature on Kedro-Viz enables seamless sharing of pipeline
 
 3. **Publish and Share Kedro-Viz with Github Actions**: Use the `publish-kedro-viz` action available on the Github Marketplace to deploy Kedro-Viz for your Kedro project repository on GitHub Pages. For further information and usage instructions, please refer to [publish-kedro-viz](https://github.com/marketplace/actions/publish-kedro-viz)
 
+## Filtering and sharing Kedro-Viz pipelines 
+
+Kedro-Viz allows user to focus on specific aspects of their data pipeline by filtering the visualization. Once filters are applied, the resulting visualization can be shared with others by sharing the URL
```
Comment (astrojuanlu) [inline docs/source/share_kedro_viz.md:16]:
```suggestion
Kedro-Viz allows you to focus on specific aspects of your data pipeline by filtering the visualization. Once filters are applied, you can share the resulting visualisation with others by copying the URL
```

?

File: docs/source/share_kedro_viz.md [hunk 2]
```diff
@@ -11,6 +11,16 @@ The publish and share feature on Kedro-Viz enables seamless sharing of pipeline
...
+
+As filters are applied, the URL in the browser automatically updates to reflect the current state of your filtered pipeline. By selecting filters like type, tags, or focus mode through the user interface, users generate a URL that can be shared, granting others access to the same filtered view of the pipeline.
```
Comment (astrojuanlu) [inline docs/source/share_kedro_viz.md:18]:
```suggestion
As filters are applied, the URL in the browser automatically updates to reflect the current state of your filtered pipeline. By selecting filters like type, tags, or focus mode through the user interface, you can generate a URL that can be shared, granting others access to the same filtered view of the pipeline.
```

File: docs/source/share_kedro_viz.md (hunk 1 above)
Comment (rashidakanchwala) [reply docs/source/share_kedro_viz.md:16]: lol i had it as you before and then i changed the tone to 'users' :D

Comment (astrojuanlu) [reply docs/source/share_kedro_viz.md:16]: I usually follow https://ddbeck.com/second-person-is-ok/ and https://github.com/kedro-org/kedro/wiki/Kedro-documentation-style-guide#style

**Review by jitu5 — APPROVED** [review]

**Review by astrojuanlu — APPROVED** [review]

**Review by stephkaiser — APPROVED** [review]
looks good! thanks team

**Review by Huongg — APPROVED** [review]
LGTM 👍

**Pull request merged**